client/post-editor/editor-ground-control/index.jsx

```diff
@@ -399,6 +479,17 @@ export default React.createClass( {
 						this.schedulePostPopover()
 					}
 				</div>
+				{
+					this.state.needsVerification &&
+					<div className="editor-ground-control__email-verification-notice" tabIndex={ 7 } onClick={ this.props.onMoreInfoAboutEmailVerify }>
+						<Gridicon
+							icon="info"
+							className="editor-ground-control__email-verification-notice-icon" />
+						{ this.getVerificationNoticeLabel() }
+						{ ' ' }
+						<u>{ this.translate( 'Learn More' ) }</u>
```

Comment:
I was worried about the potential accessibility implications of using an  `<a>` or a  `<button>` element nested within an already clickable element (The entire notice is clickable, not only the “Learn More” portion).

I also felt like maybe adding a `<span>` with a class, and styling that would be overkill, that's why I went with `<u>`. I'm going to fix this on #6230.

Now that I think about it, we might need an `aria-role` for the notice, right?